.github/ISSUE_TEMPLATE.md

```diff
@@ -6,14 +6,17 @@ A summary of the issue.
 
 ### Steps to Reproduce
 
-1. This is the first step
-2. This is the second step, etc.
+1. (for example) Start a new project with `wagtail start myproject`
+2. Edit models.py as follows...
+3. ...
+
+(It's essential that you provide enough information for someone else to replicate the problem you're seeing. Simply describing something that's broken on your current project is not enough!)
 
 Any other relevant information. For example, why do you consider this a bug and what did you expect to happen instead?
 
 ### Technical details
 
 * Python version: Run `python --version`.
 * Django version: Look in your requirements.txt, or run `pip list | grep -i django`.
-* Wagtail version: Hover over the Wagtail bird in the admin, or look in the Settings menu, look in your requirements.txt, or run `pip list | grep -i wagtail`.
+* Wagtail version: Hover over the Wagtail bird in the admin, or run `pip list | grep -i wagtail`.
```

Comment:
How much work would it be to create a dedicated command line tool (`wagtail doctor`) or page in the admin interface that gathers all of this info directly?